### ISSUE TITLE
User Id is generating #2 again

### ISSUE THREAD

**Issue by ClayC90** [StrangeLoopGames/EcoIssues#8263]
Went in game using my steam version of the game, when I used the SLG client, it didn't recognise me and called me ClayC#2.

Commits related to this issue were already done and for the most part it does work well, just not in the above scenario.

**Comment by MarjieAVolk** [comment]
Are your steam and slg accounts linked?

**Comment by MarjieAVolk** [comment]
They may have gotten unlinked a few months ago when we were having issues with our database, so you might want to double check.

**Comment by ClayC90** [comment]
Yes they are linked, I get both IDs when I run /whoami from the steamed version and the website says that they are linked, I can re-link and re-test if you like.

**Comment by MarjieAVolk** [comment]
No that's fine, if /whoami shows you both, it shouldn't create a second user.

Can you check /whoami on both your original user and your #2 user?  Are they different?

**Comment by ClayC90** [comment]
How do I check that correctly please?

**Comment by MarjieAVolk** [comment]
Log in via steam, as 'ClayC' and type '/whoami'.  Take a screenshot of it.  Then log in via the web page download as 'ClayC#2' and type '/whoami'.  Take a screenshot of that.  And share those here, or just compare them yourself if you don't want to put that on the internet.
